<launch>
  <node pkg="multirobot_detect" name="multirobot_detect_node" type="multirobot_detect_node" output="screen">
  <param name="subscribed_topic" type="string" value="/dji_sdk/image_raw"/>
  <param name="show_video_flag" type="bool" value="true"/>
  <param name="save_result_video_flag" type="bool" value="true"/>
  <param name="result_video_file_name" type="string" value="/home/ubuntu/ros_my_workspace/src/multirobot_detect/result/a544.avi"/>
  <param name="rate" type="double" value="20.0"/>
  <param name="image_hight" type="double" value="480.0"/>
  <param name="image_width" type="double" value="640.0"/>
  <param name="save_set_flag" type="bool" value="false"/>
  </node>
</launch>
